### PR TITLE
Bridge Re-mapped

### PR DESCRIPTION
**Pull request by AuroraBirdie**
Moved the weapons console forward
Moved Sensors and Thrust Consoles to a side bay.
More table space (for that office feel)
Added a Captain's workstation where the pillar use to be Moved cameras to side pillars
Intercomms given to each workstation
Medical/Security consoles moved to back left
Message monitoring and alerts moved by doors leading onto bridge
image
!review required

### CODE REVIEW

**Comment by AuroraBirdie** [comment]
image
Forgot about the two plants I put by the walls

**Comment by Greenjoe12345** [comment]
Not all that keen on the main control consoles being separated

**Comment by AuroraBirdie** [comment]
[AuroraBirdie-Bridge-Remap](https://github.com/AuroraBirdie/Aurora.3/tree/AuroraBirdie-Bridge-Remap)

Took feedback from [This post](https://forums.aurorastation.org/topic/18142-bridge-helm-re-map/?do=findComment&comment=163925) removed tables, moved shredder, put medical, cam, and alert console together. Put Messaging console by itself. Added table to Gunner station, moved sensor console over.

**Comment by Eyeveri** [comment]
> Not all that keen on the main control consoles being separated

I agree with this. Secondary consoles would be fine but, I do not like that the Sensors, Engine control and nav console is all separate.

**Comment by FluffyGhoster** [comment]
> Not all that keen on the main control consoles being separated


> > Not all that keen on the main control consoles being separated
> 
> I agree with this. Secondary consoles would be fine but, I do not like that the Sensors, Engine control and nav console is all separate.

Feedback thread is in https://forums.aurorastation.org/topic/18142-bridge-helm-re-map/
Please remember that Github is only about the code (or the map, in this case) itself

**Comment by AuroraBirdie** [comment]
Helm/Sensors/Thrusters moved all back together. See below and changelog.
image

**Comment by AuroraBirdie** [comment]
image
Fixed 'ssssss' ref. var. issue, moved things for 'flavor' which does not change the changelog

**Comment by AuroraBirdie** [comment]
image
Consoles couldn't be rotated, no sprites to change with. Removed Intercom in bottom picture.

image
https://discord.com/channels/157516682288562176/302201161044328450/1087442344657305660

**Comment by AuroraBirdie** [comment]
image
Alright last time I plan to touch it for a while. Will gather some feedback for it and hopefully it makes it's way onto the horizon.

**Comment by AuroraBirdie** [comment]
!review required

**Comment by AuroraBirdie** [comment]
Map keeps breaking. I can't fix it. Read can't seem to fix it, this is so wild, honestly. So moving my thing to a suggestion instead :/ I don't know how to fix this, even when redoing it by hand with the most recent map.

File: html/changelogs/AuroraBirdie-PR-#16055.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  - maptweak: "removed intercomms to old bridge crew stations. Put intercomm by sensor console"
+  - maptweak: "added extra chair for seating choices and slight requirements"
+  - maptweak: "added pillar behind captain's chair for a camera and lights"
+  - maptweak: "added extra vent"
+  - maptweak: "Hid all vents and scrubbers"
+  - maptweak: "added plants by consoles"
+  - maptweak: "removed one of the paperbins/folders/pen sets"
+  - maptweak: "messaging/engineer console put in back left"
+  - maptweak: "removed all consoles from top right, as all other stations were handled. Put decorations there instead"
+  - maptweak: "added naviation console to the Captain's seat- made it more symmerical"
```
Comment (SleepyGemmy) [inline html/changelogs/AuroraBirdie-PR-#16055.yml:63]:
```suggestion
  - maptweak: "Remapped the bridge."
```
no. please shorten this changelog to a single entry.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
PR is reverting some map changes, specifically on the hull. refer to the [mapdiff](https://github.com/Aurorastation/Aurora.3/pull/16055/checks?check_run_id=12276918104).

**Comment by AuroraBirdie** [comment]
image
Fixed the hull back as that was never the point of this PR. Moved conference console back. Should be all good now, right?

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**